src/code/actions.js

```diff
@@ -257,12 +258,25 @@ function getEarnedCoinString(state) {
   return ["~ALERT.NEW_PIECE_OF_COIN", scoreString];
 }
 
-function _submitDrake(correctPhenotype, submittedPhenotype, correct) {
-  let incrementMoves = !correct;
-  return{
+function _submitDrake(targetDrakeIndex, userDrakeIndex, correct, state) {
+  const incrementMoves = !correct,
+        targetDrakeOrg = GeneticsUtils.convertDrakeToOrg(state.drakes[targetDrakeIndex]),
+        userDrakeOrg = GeneticsUtils.convertDrakeToOrg(state.drakes[userDrakeIndex]),
+        initialDrakeOrg = GeneticsUtils.convertDrakeToOrg(state.initialDrakes[userDrakeIndex]),
+        routeSpec = state.routeSpec,
+        editableGenes = state.authoring[routeSpec.level][routeSpec.mission][routeSpec.challenge].userChangeableGenes;
+
+  return {
     type: actionTypes.DRAKE_SUBMITTED,
-    correctPhenotype,
-    submittedPhenotype,
+    species: BioLogica.Species.Drake.name,
+    correctPhenotype: targetDrakeOrg.phenotype.characteristics,
+    submittedPhenotype: userDrakeOrg.phenotype.characteristics,
+    initialAlleles: initialDrakeOrg.alleles,
+    selectedAlleles: userDrakeOrg.alleles,
+    // Target alleles contains the alleles of the target drake- there may be multiple correct allele strings
+    targetAlleles: targetDrakeOrg.alleles,
+    targetSex: targetDrakeOrg.sex,
```

Comment:
I realize that the GUIDE version didn't have `submittedSex`, but I think that must just be an oversight on their part (their little demo game didn't allow the user to change the sex, so they probably forgot). Maybe it should be added?